bin/juxtaposer/deploy/secretless.yml

```diff
@@ -0,0 +1,84 @@
+listeners:
```

Comment:
are you planning to upgrade to the v2 config?

Reply:
indeed it's just that the broker didn't have the v2 config merged yet